backend/curriculum_tracking/tests/test_activity_log_entry_creators.py

```diff
@@ -155,8 +155,31 @@ def test_start_project(self):
 # class log_card_review_request_cancelled_Tests(TestCase):
 
 
-# class log_card_moved_to_complete_Tests(TestCase):
+class log_card_moved_to_complete_Tests(TestCase):
+    LIST_URL_NAME = "agilecard-list"
+    SUPPRESS_TEST_POST_TO_CREATE = True
```

Comment:
In you are4n't inheriting from the api test mixin then these lines aren't necessary